templates/3a_monitor_MIBI_run.ipynb

```diff
@@ -29,8 +29,8 @@
    "outputs": [],
```

Comment:
<p>This shows up as on a newline in the actual notebook: https://github.com/angelolab/toffy/blob/7238168e8702d19bb0a2358a3ed66f5b49a4f280/templates/3a_monitor_MIBI_run.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>